src/kudu/tserver/simple_tablet_manager.cc

```diff
@@ -277,48 +276,34 @@ Status TSTabletManager::WaitUntilRunning() {
   return Status::OK();
 }
 
-Status TSTabletManager::InitRaftAsync(bool is_first_run) {
-  if (is_first_run) {
-    RETURN_NOT_OK(CreateNew(server_->fs_manager()));
-  } else {
-    RETURN_NOT_OK(Load(server_->fs_manager()));
-  }
-  return Status::OK();
+bool TSTabletManager::IsInitialized() const {
+  return state() == MANAGER_INITIALIZED;
```

Comment:
Looks like the new status is more for posterity. Ignore my comment on the proto file.